top-crates/Cargo.toml

```diff
@@ -4,14 +4,11 @@ name = "top-crates"
 version = "0.1.0"
 
 [dependencies]
+cargo = "0.16"
 hyper = "0.10.4"
 hyper-native-tls = "0.2.2"
-mktemp = "0.3.0"
-semver = "0.6.0"
+lazy_static = "0.2"
 serde = "0.9.6"
+serde_derive = "0.9"
 serde_json = "0.9.5"
-
-[dependencies.toml]
-version = "0.2.0"
-default-features = false
-features = ["serde"]
+toml = { git = "https://github.com/alexcrichton/toml-rs" }
```

Comment:
Serde 0.9 support exists on the master branch, update to 0.3 once it is available. https://github.com/alexcrichton/toml-rs/pull/137#issuecomment-278647851